src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -18,22 +21,22 @@ public String index(ModelMap map) {
 
     @Autowired
```

Comment:
Moved both autowired repositories to the top of the file as they can be used by any method